test/evergreen.yml

```diff
@@ -2987,8 +3011,8 @@ tasks:
           # Set this in case of a core to get the correct python binary.
           python_binary: $(pwd)/venv/bin/python3
           tiered_storage_test_name: tiered06
-  - name: tiered-unittest-test
-    tags: ["pull_request"]
+  - name: s3-tiered-unittest-test
```

Comment:
Thanks for fixing these names, it always bothered me a tiny bit, but never enough to make a ticket + PR.